doc/api/next_api_changes/behaviour.rst

```diff
@@ -36,3 +36,8 @@ did nothing, when passed an unsupported value. It now raises a ``ValueError``.
 `.Axis.set_tick_params` (and the higher level `.axes.Axes.tick_params` and
 `.pyplot.tick_params`) used to accept any value for ``which`` and silently
 did nothing, when passed an unsupported value. It now raises a ``ValueError``.
+
+``font_manager.findfont`` now warns of +-100 font-weight difference of selected font vs found font
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
our docs are picky about the underline length matching the text line.  My attempt to suggest the change failed due to github's text input being non-mono-spaced.